data/human/deep missions.txt

```diff
@@ -2471,29 +2471,63 @@ mission "Stone of our Fathers 1"
 		conversation
 			`Helheim's spaceport has all the constant hustle and bustle of a boomtown. It's dirty, lively, and packed with workers. As you walk through the spaceport, an aged man in a worn and weathered roustabout jacket waves his arm toward you and approaches you.`
 			`	"You've got the look of one who knows a ship inside and out," he says. "Name's Skaldgar, ain't much for ships; earth, soil, and rock are my domain. But I do got a need to travel." He pauses as a wistful look briefly passes over his face. "I left, you see, decades ago. No goodbye. No note. Just... left. Living with others, taking care of others, hard on a man like me. I like solitude, peace, and work. But now my bones are brittle: the radiation, earth-rot as we call it, has taken hold and my time is running out. And I find, I don't want to be alone anymore."`
-			`	He stops and frowns. "Look at me, rambling like an ol' man. Captain, I'd like to pay you <payment> to take me home, to <destination>."`
+			`	He stops and frowns. "Look at me, rambling like an ol' man. Ser, Captain, I'd like to pay you <payment> to take me home, to <destination>."`
```

Comment:
Leaving "sir" prompts the reason for asking the question.

Reply:
But he doesn't call you sir.

Reply:
I meant change it to 'sir' before the explanation and then 'ser' after. We read it as 'sir' even if it's spelt 'ser'.  I think it might help the player understand that 'sir' and 'ser' sound the same but are indeed two different words.

Reply:
I understand. But that is not what he says; he does not call you sir. 

You want him to call you 'sir' so you can have a reason for the player to ask, 'why did you call me sir'. But he doesn't call you that. This is very much putting the cart before the horse and it creates all kinds of additional problems. 

You're forcing the player to be confused and to hear him incorrectly. You're removing agency from the player, and you're also damaging the game's credibility - since the game is literally misleading the player here.

And all of this just feel unnecessary. Skaldgar says 'ser', let him say what he says, why lie/mislead/force the player to mishear him? If the player is confused they can ask "why did you call me ser?" and Skaldgar then explains its a term of respect.

Reply:
Nobody is mishearing anything , both words sound the same, so it makes sense that the player would spell it _sir_. IMO Spelling it _ser_ puts the cart before the horse because the player doesn't know anything about _ser_ yet.

If the players character is a male then _sir_ would make sense and there is no confusion but the player might wonder why the "why are you calling me sir" question is there and click to find out. However, if the players character is not male then the "why are you calling me sir" question makes sense and leads to the reveal that _ser_ is actually a different word and is what was meant.